object.go

```diff
@@ -346,7 +346,7 @@ func (d *Decoder) decInstance(typ reflect.Type, cls classInfo) (interface{}, err
 					d.unreadByte() // Enum parsing, decInt64 above has read a byte, so you need to return a byte here
 					s, err := d.DecodeValue()
 					if err != nil {
-						return nil, perrors.Wrapf(err, "decInstance->decObject field name:%s", fieldName)
+						return nil, perrors.Wrapf(err, "decInstance->Deserialize field name:%s", fieldName)
```

Comment:
I think it doesn't needed to change the comment.

Reply:
Isn't the name consistent?